setup.py

```diff
@@ -2,13 +2,16 @@
 Setup script.
 """
 
+import io
+from os import path
 from setuptools import setup, find_packages
 
 if __name__ == '__main__':
+    this_directory = path.abspath(path.dirname(__file__))
     with \
             open('requirements.txt') as requirements, \
             open('test_requirements.txt') as test_requirements, \
-            open('README.md') as readme:
+            io.open(path.join(this_directory, 'README.md'), encoding='utf-8') as readme:
```

Comment:
Why the use of `this_directory` here but not above?

Reply:
I assumed nothing needs to be changed for the requirements files, but I guess you could add the same changes there as well.

Reply:
I mean, why is `this_directory` needed at all?

Reply:
Maybe it's not - i'm not sure. Feel free to change it.

Reply:
It’s not in the sense that `setup.py` is supposed to always be run from its parent directory.